core/test/qa_Scheduler.cpp

```diff
@@ -208,50 +243,85 @@ checkBlockNames(const std::vector<TBlock> &joblist, std::set<std::string> set) {
     }
 }
 
+template<typename T>
```

Comment:
Since this class is useful also for other unit-tests, it should be moved/declared in the blocklib (alongside the other test blocks).

Mind the naming convention and exported public member field names (snake_case)

Reply:
Generally, we should aim to have most of the unit tests reuse the same test block definitions.

Reply:
I will extend it in the next "LifeCycle" PR and move to the blocklib.